src/js/commentCounter.js

```diff
@@ -0,0 +1,33 @@
+const commentsEndpoint = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8v2YvLQLsPQiil6nHJBM/comments';
+const fetchMovieComments = async (movieId) => {
+  const response = await fetch(`${commentsEndpoint}?item_id=${movieId}`).catch((err) => err);
+  return response.json();
+};
+
```

Comment:
[Optional]
- [ ] You did a great work with putting all (fetchData, addLikes, getLikes) API related methods in `api.js`, but looks like API queries (fetchMovieComments, and addComment in `CommentPopup.js`) for comments located in different files which make the project structure a little bit odd. Naturally, I was expecting to see these methods in the `api.js` file also.